packages/terra-embedded-component/src/EmbeddedComponent.jsx

```diff
@@ -1,31 +1,135 @@
+/* global SystemJS */
 import React from 'react';
 import PropTypes from 'prop-types';
-import classNames from 'classnames/bind';
-import 'terra-base/lib/baseStyles';
-import styles from './EmbeddedComponent.scss';
+import axios from 'axios';
+import 'systemjs/dist/system-production';
+import { loadStylesheet } from './assets';
 
-const cx = classNames.bind(styles);
+// Expose AMD required globals on the window
+window.define = window.SystemJS.amdDefine;
+window.require = window.requirejs = SystemJS.amdRequire;
```

Comment:
I don't think this is requried so I'm wondering if we should remove this line (define is still needed)

Reply:
Got rid of this line with latest commit